go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/osbuild/image-builder
 
-go 1.21
+go 1.22.1
```

Comment:
So for the go version we pick whatever is available in the registry.access.redhat.com/ubi9/go-toolset container. Which is currently `1.21.10`.